### ISSUE TITLE
Is it possible offline navigation if you download Mapbox offline

### ISSUE THREAD

**Issue by lalit-suthar-dev** [homeeondemand/react-native-mapbox-navigation#94]
Is it possible offline navigation if you download Mapbox offline. Is there any other way for offline navigation for Mapbox?

**Comment by KindCoder-no** [comment]
I dont think so. I do believe that mapbox requires network connection to manage api request count (limit set by mapbox)
